CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
+- Fix: avoid duplicated attributes and metadata in arrays in POST /v2/subscriptions (#2101)
```

Comment:
```suggestion
- Fix: avoid duplicated attributes and metadata in arrays in POST /v2/subscriptions and PATCH /v2/subscription/{subId} (#2101)
```

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/4025/commits/dc2c8bbc49fc88cb4a790524abd1a86a01dd1387